django-resonant-settings/resonant_settings/development/celery.py

```diff
@@ -1,7 +1,13 @@
+from resonant_settings._env import env
+
 # Acknowledge early in development, which will help prevent failing or
 # long-running tasks from being started automatically every time the worker
 # process restarts; this more aggressively flushes the task queue.
 CELERY_TASK_ACKS_LATE = False
 
+# Expose task_always_eager configurability via the environment for debugging purposes.
+CELERY_TASK_ALWAYS_EAGER = env.bool("DJANGO_CELERY_TASK_ALWAYS_EAGER", default=False)
```

Comment:
```suggestion
CELERY_TASK_ALWAYS_EAGER: bool = env.bool("DJANGO_CELERY_TASK_ALWAYS_EAGER", default=False)
```
django-environ doesn't expose usable types.